app/routers/profile.py

```diff
@@ -1,23 +1,22 @@
 import io
 
 from fastapi import APIRouter, Depends, File, Request, UploadFile
+from PIL import Image
 from starlette.responses import RedirectResponse
 from starlette.status import HTTP_302_FOUND
-from PIL import Image
 
 from app import config
 from app.database.database import get_db
 from app.database.models import User
 from app.dependencies import MEDIA_PATH, templates
 
-
 PICTURE_EXTENSION = config.PICTURE_EXTENSION
 PICTURE_SIZE = config.AVATAR_SIZE
 
 router = APIRouter(
     prefix="/profile",
     tags=["profile"],
-    responses={404: {"description": "Not found"}},
+    responses={404: {"description": _("Not found")}},  # noqa F821
```

Comment:
How does the `_` function work here if it isn't imported?

Reply:
I've now read your comments. Interesting. Can you add the flake configuration file? (You can also add the changes directly to the workflow, editing the flake8 cli parameters)

Reply:
Should I add a tox.ini file like they give in their project [here](https://flake8.pycqa.org/en/latest/user/configuration.html), or should I edit python-app.yml? If it's the latter, how do I push changes to it as I don't have the workflow folder in my project.